.github/CHANGELOG.md

```diff
@@ -216,143 +209,105 @@
-  object implements the operations of a field. See tests/tape/test_jacobian_tape.py
-  for an example.
+  tapes, fn = qml.transforms.hamiltonian_expand(tape)
+  ```
 
-* PennyLane now has a ``kernels`` module.
-  It provides basic functionalities for working with quantum kernels as well as
-  post-processing methods to mitigate sampling errors and device noise:
-
-```python
-import pennylane as qml
-from pennylane import numpy as np
-
-num_wires = 6
-wires = range(num_wires)
-
-dev = qml.device('default.qubit', wires=num_wires)
-
-@qml.qnode(dev)
-def kernel_circuit(x1, x2):
-    qml.templates.AngleEmbedding(x1, wires=wires)
-    qml.adjoint(qml.templates.AngleEmbedding)(x2, wires=wires)
-    return qml.probs(wires)
-
-kernel = lambda x1, x2: kernel_circuit(x1, x2)[0]
-
-# "Training feature vectors"
-X_train = np.random.random((10, 6))
-# Create symmetric square kernel matrix (for training)
-K = qml.kernels.square_kernel_matrix(X_train, kernel)
-# Add some (symmetric) Gaussian noise to the kernel matrix.
-N = np.random.randn(10, 10)
-K += (N + N.T) / 2
-
-K1 = qml.kernels.displace_matrix(K)
-K2 = qml.kernels.closest_psd_matrix(K)
-K3 = qml.kernels.threshold_matrix(K)
-K4 = qml.kernels.mitigate_depolarizing_noise(K, num_wires, method='single')
-K5 = qml.kernels.mitigate_depolarizing_noise(K, num_wires, method='average')
-K6 = qml.kernels.mitigate_depolarizing_noise(K, num_wires, method='split_channel')
-
-# "Testing feature vectors"
-X_test = np.random.random((5, 6))
-# Compute kernel between test and training data.
-K_test = qml.kernels.kernel_matrix(X_train, X_test, kernel)
-```
+  We can now evaluate the transformed tapes, and apply the post-processing
+  function:
 
-* QNodes now display readable information when in interactive environments or when printed.
-  [(#1359)](https://github.com/PennyLaneAI/pennylane/pull/1359).
+  ```pycon
+  >>> dev = qml.device("default.qubit", wires=3)
+  >>> res = dev.batch_execute(tapes)
+  >>> fn(res)
+  3.999999999999999
+  ```
 
-* Added CPhase operation as an alias for ControlledPhaseShift operation
-  [(#1319)](https://github.com/PennyLaneAI/pennylane/pull/1319).
+* The `quantum_monte_carlo` transform has been added, allowing an input circuit to be transformed
+  into the full quantum Monte Carlo algorithm.
+  [(#1316)](https://github.com/PennyLaneAI/pennylane/pull/1316)
 
-* The `qml.Toffoli` operation now has a decomposition over elementary gates.
-  [(#1320)](https://github.com/PennyLaneAI/pennylane/pull/1320)
+  Suppose we want to measure the expectation value of the sine squared function according to
+  a standard normal distribution. We can calculate the expectation value analytically as
+  `0.432332`, but we can also estimate using the quantum Monte Carlo algorithm. The first step is to
+  discretize the problem:
 
-* Added a new noise channel, `qml.ResetError`.
-  [(#1321)](https://github.com/PennyLaneAI/pennylane/pull/1321).
+  ```python
```

Comment:
I think we can reduce the size of this example by avoiding using saved constants:

```python
# The first step is to discretize the problem
import numpy as np
from scipy.stats import norm

xs = np.linspace(-np.pi, np.pi, 2 ** 5)
probs = np.array([norm().pdf(x) for x in xs])
probs /= np.sum(probs)

func = lambda i: np.sin(xs[i]) ** 2
r_rotations = np.array([2 * np.arcsin(np.sqrt(func(i))) for i in range(2 ** 5)])
```
```python
# The quantum_monte_carlo transform can then be used
import pennylane as qml
from pennylane.templates.state_preparations.mottonen import (
    _uniform_rotation_dagger as r_unitary,
)

def fn():
    qml.templates.MottonenStatePreparation(np.sqrt(probs), wires=range(5))
    r_unitary(qml.RY, r_rotations, control_wires=range(5)[::-1], target_wire=5)

dev = qml.device("default.qubit", wires=12)

@qml.qnode(dev)
def qmc():
    qml.quantum_monte_carlo(fn, wires=range(6), target_wire=5, estimation_wires=range(6, 12))()
    return qml.probs(range(6, 12))

phase_estimated = np.argmax(qmc()[:int(2 ** 6 / 2)]) / 2 ** 6
print((1 - np.cos(np.pi * phase_estimated)) / 2)
```

Reply:
I have to admit, I'm a bit confused as to why a private function `_uniform_rotation_dagger` needs to be imported?

Reply:
Thanks! It is indeed shorter, though the example also seems to be a bit less readable overall. Would we like to target brevity here?

As for the private function, this seems to be a function used for QMC frequently. The example was added in 8d28326d03eee18cf3de570dac9f8ba3fa17fe64 following the [QMC example from the Usage Details](https://pennylane.readthedocs.io/en/latest/code/api/pennylane.quantum_monte_carlo.html?highlight=quantum_monte_carlo#pennylane.quantum_monte_carlo). Would we like to update that?